### PR TITLE
feat - added toast component

### PR DESCRIPTION
**Pull request by ashishnnnnn**
Added Toast with cancel button

### CODE REVIEW

**Review by AmishMishra11 — COMMENTED** [review]
Looks good

File: toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.toast {
+  width: 18rem;
+  border: 2px solid #bb4430;
+  padding: 0.5rem 1rem;
+  display: flex;
+  flex-wrap: wrap;
+  word-break: break-all;
+  align-items: center;
+  justify-content: space-between;
+  border-radius: 5px;
```
Comment (shubhamkumar648) [inline toast/toast.css:10]: use relative unit for more responsive

Comment (ashishnnnnn) [reply toast/toast.css:10]: Yaa.. we should use rem.. But this was for border.. so i given it in px

Comment (shubhamkumar648) [reply toast/toast.css:10]: > Yaa.. we should use rem.. But this was for border.. so i given it in px

ok fine

**Pull request merged**